stylesheets/documentation.css

```diff
@@ -455,6 +454,50 @@ pre {
     font-size: large;
 }
 
+/* Navigation */
+.top-nav-wrapper{
+    background: var(--SIMPLE-ALERTS-SECTION)
+}
+
+.top-nav{
+    padding: 1rem;
+    background: var(--SECONDARY-COLOR);
+    border-radius: 4px;
+    width: 100%;
```

Comment:
I think max width and width will do the same thing here

Reply:
Thank you, I'll check that.